lua/telescope/_extensions/project/utils.lua

```diff
@@ -64,7 +64,7 @@ end
 
 -- Parses path into project object (activated by default)
 M.get_project_from_path = function(path)
-    local title = path:match("[^/]+$")
+    local title = tostring(path):match("[^/]+$")
     local activated = 1
     local line = title .. "=" .. path .. "=" .. activated
```

Comment:
For some reason I'm seeing `line` printed to messages on startup, running `tostring(path)` seems to fix that.

Reply:
This PR also needs https://github.com/nvim-lua/plenary.nvim/pull/202